client/me/purchases/components/payment-method-form/helpers.js

```diff
@@ -0,0 +1,229 @@
+/**
+ * External dependencies
+ */
+import { useState, useEffect, useRef } from 'react';
+import { camelCase, kebabCase, debounce } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import notices from 'calypso/notices';
+import {
+	handleRenewNowClick,
+	isRenewable,
+	shouldAddPaymentSourceInsteadOfRenewingNow,
+} from 'calypso/lib/purchases';
+import wpcomFactory from 'calypso/lib/wp';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+
+const wpcom = wpcomFactory.undocumented();
+
+export async function saveOrUpdateCreditCard( {
```

Comment:
Since all these functions are just being duplicated without any changes (I think; correct me if I'm wrong), we could change the "old" code to pull these functions from this file instead. It might not be worth it, but I'm curious what you think.

My reasoning is that we're working on the logic around these functions in https://github.com/Automattic/wp-calypso/pull/47642 and if we have a duplicate copy here, we're going to need to make sure we make changes in both places. I'm worried that something will be fixed in the old location and not fixed here, and then we'll eventually remove the old location with all its fixes.

Reply:
> we could change the "old" code to pull these functions from this file instead.

Yeah, that makes sense to me. My thinking was that we would delete the old file once we were ready with this one.